src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -421,9 +421,11 @@ const WorkflowView = _.flow(
       ])
       const { methodConfiguration: config } = validationResponse
       const inputsOutputs = await Methods.configInputsOutputs(config)
+      const methods = await Methods.list()
```

Comment:
Also unsure if this is the best place to do this methods list call/save to state